src/daft-dsl/src/expr.rs

```diff
@@ -131,7 +137,8 @@ impl AggExpr {
             | Min(expr)
             | Max(expr)
             | List(expr)
-            | Concat(expr) => expr.clone(),
+            | Concat(expr) => vec![expr.clone()],
+            MapGroups { func: _, inputs } => inputs.iter().map(|e| e.clone().into()).collect(),
```

Comment:
hmmm, doing `inputs.clone()` gives an error:

`match` arms have incompatible types
expected struct `Vec<Arc<expr::Expr>>`
   found struct `Vec<expr::Expr>
   
I'll just stick to this since the impl for FunctionExpr also does the same

Reply:
Oh! Interesting didn't realize we weren't using `ExprRefs`

Makes sense.